applications/vessel-history/src/features/home/Home.tsx

```diff
@@ -7,6 +8,7 @@ import { Spinner, IconButton } from '@globalfishingwatch/ui-components'
 import { logoutUserThunk } from 'features/user/user.slice'
 import { Vessel } from 'types'
 import VesselListItem from 'features/vessel-list-item/VesselListItem'
+import SearchPlaceholder, { SearchNoResultsState } from '../search/SearchPlaceholders'
```

Comment:
In general I'd rather prefer absolutely imports instead of relative ones to easily move code if needed but this is a minor thing